chipsec/cfg/pch_c620.xml

```diff
@@ -359,6 +359,7 @@ XML configuration file for
   <controls>
     <control name="BiosInterfaceLockDown"  register="BC"    field="BILD"    desc="BIOS Interface Lock-Down"/>
     <control name="SpiWriteStatusDis"      register="HSFS"  field="WRSDIS"  desc="Write Status Disable"/>
+    <control name="ACPIBaseLock" register="GEN_PMCON_B" field="ACPI_BASE_LOCK" desc="Lock Down ACPI Base Address" />
```

Comment:
Did you mean GEN_PMCON_1?

Reply:
Yes I did, thanks for catching it

Reply:
Actually, I take that back
This patch depends on #578